website/content/api-docs/index.mdx

```diff
@@ -97,6 +97,15 @@ action if no intention matches.
 
 This is returned even if ACLs are disabled.
 
+## Results Filtered by ACLs
+
+As of Consul 1.11, most list endpoints support an `X-Consul-Results-Filtered-By-ACLs`
+HTTP response header. This indicates that the response contains a partial subset
+of results, because some have been filtered out by ACL policies.
```

Comment:
I know it could be really tedious but it may be worth calling out exactly which endpoints support this today/ as of this change.

Reply:
Yup! I appreciate that "most" is a bit of a [weasel word](https://en.wikipedia.org/wiki/Weasel_word) here 😄 do you know of anywhere else in the docs where we list endpoints like this?

I guess an alternative would be to call it out on the endpoint docs themselves?